packages/react-components/src/CopyButton.tsx

```diff
@@ -33,8 +34,13 @@ function CopyButton ({ children, className, icon = 'copy', isAddress = false, va
         message: t<string>('address copied'),
         status: 'queued'
       });
+      isMnemonic && queueAction && queueAction({
+        action: t<string>('clipboard'),
+        message: t<string>('mnemonic copied'),
+        status: 'queued'
+      });
```

Comment:
This can just be combined with the one above, just do the `?:` alongside the message.

Reply:
Do you mean sth like:
```
      (isAddress || isMnemonic) && queueAction && queueAction({
        account: value,
        action: t<string>('clipboard'),
        message: t<string>(`${isAddress ? 'address' : 'mnemonic'} copied`),
        status: 'queued'
      });
```
?

Reply:
I missed this question originally, sorry about that. Yes.